machine/machine.go

```diff
@@ -191,6 +193,45 @@ func getNodeIdFromCpuBus(cpuBusPath string, threadId int) (int, error) {
 	return nodeId, nil
 }
 
+// GetHugePagesInfo returns information about pre-allocated huge pages
+func GetHugePagesInfo(hugepagesDirectory string) ([]info.HugePagesInfo, error) {
```

Comment:
Maybe add a comment about what directories this function wants.

Reply:
nit: This makes it easy to write some simple tests. 

There are already some tests her: https://github.com/google/cadvisor/blob/master/machine/topology_test.go. The coverage of unit tests is low, but some tests never hurt I guess. :smile:

Reply:
I added unit test and comment